src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { UserStatus } from "types/User";
+import { useShowHide } from "hooks/useShowHide";
+
+import "./UserStatusDropdown.scss";
+
+export interface UserStatusDropdownProps {
+  options: Array<UserStatus>;
+  onChange: (option: UserStatus) => void;
+  label: UserStatus | string;
+}
+
+export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
+  options,
+  label,
+  onChange,
+}) => {
+  const { isShown, hide, toggle } = useShowHide();
```

Comment:
assign understandable var names.

```suggestion
  const { isShown: isDropdownOptionsShown, hideDropdownOptions, toggle: toggleDropdownOptions } = useShowHide();
```